lib/ansible/plugins/connection/paramiko_ssh.py

```diff
@@ -420,7 +472,7 @@ def _connect_uncached(self):
 
         ssh.set_missing_host_key_policy(MyAddPolicy(self._new_stdin, self))
 
-        conn_password = self.get_option('password') or self._play_context.password
+        conn_password = self.get_option('password') or self.get_option('password')
```

Comment:
This doubles up here

```suggestion
        conn_password = self.get_option('password')
```